packages/wonder-blocks-tokens/src/theme/semantic/semantic-color-thunderblocks.ts

```diff
@@ -119,6 +119,7 @@ const core = {
     },
     shadow: {
         transparent: transparent,
+        instructive: `color-mix(in srgb, ${color.blue_10} 20%, ${transparent})`,
```

Comment:
Will this token also be added in Figma? The color tokens in the handshake file doesn't include this token! Though it seems the transparent token for TB has a blueish tint? 

https://www.figma.com/design/e9qdyiDUBZ3rDabP9S7ulO/%E2%9A%A1%EF%B8%8F-Handshake?node-id=3040-15249&m=dev

Reply:
oh good catch! turns out that I was using the incorrect tokens and incorrect values. I fixed that by moving it to `core.shadow.transparent`, and using the correct value in the modal component-level `dialog.shadow.default` token. It should now match the designs as expected. Let me know what you think.